index.html

```diff
@@ -22,6 +16,12 @@
 etherpad:         # optional (insert the URL for your Etherpad if you're using one)
 eventbrite:       # optional (insert the alphanumeric key for Eventbrite registration, e.g., "1234567890AB")
 ---
+
+
+<!-- See instructions in the comments below for how to edit specific sections of this workshop template. -->
```

Comment:
Hmm, or maybe that is supposed to just be about the YAML front matter.  Maybe leave this comment here and move the HEADER comment out of the HTML and into the YAML front matter?

    ---
    # Edit the values in this block to be appropriate…
    layout: workshop  # DON'T CHANGE THIS.
    …
    ---
    <!-- See instructions in the comments below … -->

    <!--
      EVENTBRITE
      …